README.md

```diff
@@ -7,25 +7,168 @@ An engineer working for a [CSE](./CSE.md) project...
 * Leads by example. Models the behaviors we desire both interpersonally and technically (this is what most people think of as a "developer").
 * Strives to understand how their work fits into a broader context and ensures the outcome.
 
-This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
-
-## The Basics
```

Comment:
We still need the basics as this is the minimum bar for all projects. Having them as a list at the top of the readme is crucial. In addition, each described fundamental should follow the following pattern:
+Goals
+Evidence and Measures
+General Guidance
+Resources

Reply:
See https://github.com/microsoft/code-with-engineering-playbook/pull/67#issuecomment-537187987.